app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -96,7 +99,7 @@ def setup_form
 
       # Find or create the permission_template object for this admin set
       def permission_template
-        PermissionTemplate.find_or_create_by(admin_set_id: @admin_set.id)
+        PermissionTemplate.find_or_create_by(admin_set_id: @admin_set.id, workflow_name: @workflow_name)
```

Comment:
Yeah - I think this is probably not needed at all as it's blank when created anyway.